apps/costcontrol/js/views/telephony.js

```diff
@@ -112,8 +112,10 @@ var TelephonyTab = (function() {
       billingCycle.hidden = true;
     } else {
       billingCycle.hidden = false;
-      var content = Formatting.getFormattedDate(settings.nextReset,
-        _('short-date-format'));
+      var content = Formatting.getFormattedDate(settings.nextReset, {
```

Comment:
What about exposing the object replacing `_('short-date-format')` as a getter in the `Formatting` object? For instance:

```js
Formatting.getFormattedDate(settings.nextReset, Formatting.formats.shortDate)
```